routes/users.js

```diff
@@ -0,0 +1,76 @@
+const express = require('express');
+const router = express.Router();
+const knex = require('../database');
+
+router.get('/', (req, res) => {
+  res.send('smoking tests');
+});
+
+router.get('/:user_id', (req, res) => {
+  knex.raw('SELECT * FROM users WHERE id = ?', [req.params.user_id]).then((result) => {
+    res.send(result.rows);
```

Comment:
Instead of sending an array of just one object, it's best to just return the object itself.